src/test/skript/README.md

```diff
@@ -12,6 +12,9 @@ to keep in mind:
 * Write descriptive assert messages and write comments
 * Ensure your tests pass with all supported Skript versions
   * Use standard condition for checking MC version
+* When writting tests for your pull requests, please ensure you're cleaning up afterwards. This can mean clearing your cow you spawned, or resetting the block you manipulated back to what it was. Try to avoid spawning hostile mobs.
```

Comment:
```suggestion
* When writing tests for your pull requests, please ensure that manipulations are cleaned up afterwards. This includes actions such as removing mobs that were manually spawned and resetting manipulated blocks to their original state. Also, try to avoid spawning hostile mobs.
```
Is there a justification we could include for *why* someone should avoid spawning hostile mobs. Is it related to entity despawning? Or taking damage during the day?

Reply:
The mob can target other entities, and in the case of a Zombie, it could burn from the sunlight.